runtime/oti/vmconstantpool.xml

```diff
@@ -110,6 +110,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<classref name="java/lang/invoke/MutableCallSite" flags="opt_methodHandle"/>
 	<classref name="java/lang/invoke/MethodType" flags="opt_methodHandle"/>
 	<classref name="java/lang/invoke/WrongMethodTypeException" flags="opt_methodHandle"/>
+	<classref name="java/lang/UnsupportedOperationException" flags="opt_methodHandle"/>
```

Comment:
See comment below - this should be removed or if it can't be, have a version tag added

Reply:
I have added a version tag for the above vmconstantpool entry:
```
<classref name="java/lang/UnsupportedOperationException" versions="14-"/>
```